boat-scaffold/src/main/java/com/backbase/oss/codegen/angular/BoatAngularGenerator.java

```diff
@@ -184,6 +187,15 @@ public void processOpts() {
                 log.info("  (you can select the angular version by setting the additionalProperty foundationVersion)");
             });
 
+        processOpt(SPEC_VERSION,
+            value -> additionalProperties.put(SPEC_VERSION, new SemVer(value)),
```

Comment:
Here is where it's trying to parse it as a semver (the `new SemVer(value)`. You could have an if wrapping all of this that only attempts to set the property only if `!Strings.isEmpty(value)` (which I think is nullsafe)